oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/method/annotation/OAuth2AuthorizedClientArgumentResolver.java

```diff
@@ -69,6 +73,10 @@
 	private OAuth2AccessTokenResponseClient<OAuth2ClientCredentialsGrantRequest> clientCredentialsTokenResponseClient =
 			new DefaultClientCredentialsTokenResponseClient();
 
+	private Clock clock = Clock.systemUTC();
+	private Duration accessTokenExpiresSkew = Duration.ofMinutes(1);
```

Comment:
We need to provide a way for the user to configure `accessTokenExpiresSkew`

Reply:
I saw an initial suggestion for this in a comment on the issue but when you responded that the skew already existed (as a fixed number), it looked like that addressed the concern, as the conversation went in a different direction. Are we thinking something as straightforward as a setter or did you have something else in mind?

Reply:
Actually, I take back my previous comment. We don't need `accessTokenExpiresSkew` (or `clock`) since this `HandlerMethodArgumentResolver` is not acting in the role of an OAuth Client. Only the OAuth Client needs to determine if an access token is expired before it attempts to make a protected resource request.

The only responsibility of the `HandlerMethodArgumentResolver` is to resolve the existing `OAuth2AuthorizedClient` via `OAuth2AuthorizedClientRepository` if available, and if not, than authorize the client via the configured grant type, e.g. `client_credentials`.

So let's remove `accessTokenExpiresSkew` and it's associations.